app/src/main/java/blockeq/com/stellarwallet/activities/ShowMnemonicActivity.kt

```diff
@@ -42,13 +44,22 @@ class ShowMnemonicActivity : BaseActivity(), View.OnClickListener {
     override fun onClick(v: View?) {
         val itemId = v!!.id
         when (itemId) {
-            R.id.confirmButton -> launchPINView(PinType.CREATE, getString(R.string.please_create_a_pin), mnemonicString!!, false)
+            R.id.confirmButton -> launchPINView(PinType.CREATE, getString(R.string.please_create_a_pin), mnemonicString!!, passphrase, false)
+            R.id.passphraseButton -> {
```

Comment:
Same as recovery process - button text can change to `Passphrase applied` once passphrase has been set the user.